src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionResolveHandler.cs

```diff
@@ -158,6 +185,20 @@ public async Task<CompletionItem> HandleRequestAsync(CompletionItem request, Cli
             }
 
             return resolvedCompletionItem;
+
+            static string ConvertEditIndentationToTabs(string textEdit, int tabSize)
+            {
+                var indentationString = new string(' ', tabSize);
+                var updatedTextEdit = textEdit.Replace(indentationString.ToString(), "\t");
+                return updatedTextEdit;
+            }
+
+            static string ConvertEditIndentationToSpaces(string textEdit, int tabSize)
+            {
+                var indentationString = new string(' ', tabSize);
+                var updatedTextEdit = textEdit.Replace("\t", indentationString.ToString());
+                return updatedTextEdit;
+            }
```

Comment:
Note this algorithm isn't correct if you have tabs and spaces mixed together; I don't think we'd produce those but just I don't know if we have some cases we do more complicated things.